pkg/syslog-ng-ctl/controller.go

```diff
@@ -29,6 +29,13 @@ type Controller struct {
 	lastMetricQueryTime time.Time
 }
 
+func NewController(controlChannel ControlChannel) Controller {
```

Comment:
I think you forgot to add the patch where you use this newly created constructor function.

Reply:
Indeed. Thank you.